### ISSUE TITLE
Incorrect code coverage shown for ts svelte files

### ISSUE THREAD

**Issue by juzerzarif** [mihar-22/svelte-jester#32]
Hello, I'm using svelte-jester along with svelte-preprocess to test typescript svelte files but the istanbul code coverage shows uncovered branches that don't exist in the svelte file source code.

Jest config:
```typescript
export default {
  collectCoverage: true,
  collectCoverageFrom: ['src/**/*.ts', 'src/**/*.svelte'],
  coverageDirectory: 'coverage',
  coveragePathIgnorePatterns: ['src/types'],
  coverageReporters: ['json', 'html', 'text'],
  coverageThreshold: {
    global: {
      statements: -10,
      branches: 90,
      functions: 90,
      lines: 90,
    },
  },
  moduleFileExtensions: ['js', 'ts', 'svelte'],
  moduleNameMapper: {
    '\\.html$': '<rootDir>/__mocks__/htmlMock.ts',
  },
  preset: 'ts-jest',
  roots: [process.cwd()],
  setupFiles: ['mutationobserver-shim'],
  setupFilesAfterEnv: ['./jest.setup.ts'],
  testEnvironment: 'jsdom',
  testMatch: ['**/tests/**/*.test.ts'],
  transform: {
    '^.+\\.svelte$': ['svelte-jester', { preprocess: true }],
    '^.+\\.ts$': 'ts-jest',
  },
};
```

svelte.config.js:
```javascript
const sveltePreprocess = require('svelte-preprocess');
const tailwind = require('tailwindcss');
const postcssNested = require('postcss-nested');

module.exports = {
  preprocess: sveltePreprocess({
    postcss: {
      plugins: [tailwind, postcssNested],
    },
  }),
};
```

Coverage report (html):
![image](https://user-images.githubusercontent.com/22772637/111561732-f14f8f80-8762-11eb-8fc6-0d3e1c391b79.png)

I'm assuming I've got some gaps in configuration, could someone help troubleshoot how I could get a more accurate coverage report?

**Comment by mihar-22** [comment]
Hey @juzerzarif  this is a duplicate of #28, thanks for raising!

**Comment by mihar-22** [comment]
Heads up this is fixed in `svelte-jester@1.3.1`!
